src/test/java/burrows/apps/example/template/util/AdUtilsTest.java

```diff
@@ -19,6 +19,13 @@
 public class AdUtilsTest {
 
     @Test
+    public void test_constructor() {
+        // since there is only 1 static method in the class
```

Comment:
You added these back in. I just removed these. This does not work, you made it's constructor `private`.